src/context.cc

```diff
@@ -4451,7 +4453,7 @@ void Context::syncerActivity() {
                     logger().error("Error pushing OBM action: " + err);
                 }
 
-                displayError(save(false));
+                displayError(save(false, false));
```

Comment:
I deliberately select `false` here, because it happens when the user hit "Sync" button -> Should force rendering the content again